packages/ipfs-core/src/components/pin/remote/rmAll.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const withTimeoutOption = require('ipfs-core-utils/src/with-timeout-option')
+
+module.exports = ({ serviceRegistry }) => {
+  /**
+   * Remove all pins that match the given criteria from a remote pinning service.
+   *
+   * @param {Query & AbortOptions} options
+   * @returns {Promise<void>}
+   */
+  async function rmAll (options) {
+    const { service } = options
+    const svc = serviceRegistry.serviceNamed(service)
```

Comment:
Unless I'm missing something this will return `null` unload config was loaded which may not be the case here.